code/modules/research/xenobiology/crossbreeding/_misc.dm

```diff
@@ -231,3 +231,200 @@ Slimecrossing Items
 /obj/item/capturedevice/proc/release()
 	for(var/atom/movable/M in contents)
 		M.forceMove(get_turf(loc))
+
+/obj/item/slimeball
+	name = "Slimeball"
+	desc = "Allows you to hold a single slime inside of it, just throw it into a slime, and throw again to release him!"
+	icon = 'icons/obj/slimecrossing.dmi'
+	icon_state = "slimeball"
+	var/mob/living/simple_animal/slime/held_slime
+
+/obj/item/slimeball/throw_impact(atom/hit_atom, datum/thrownthing/throwingdatum)
+	. = ..()
+	if(!isslime(hit_atom))
+		if(!held_slime)
+			return
+
+		held_slime.forceMove(get_turf(src))
+		playsound(src, 'sound/effects/splat.ogg', 40, TRUE)
+		held_slime = null
+		return
+	playsound(src, 'sound/effects/attackblob.ogg', 50, TRUE)
+	held_slime = hit_atom
+	held_slime.forceMove(src)
+
+/obj/item/spear/slime
+	name = "Slimespear"
+	desc = "Gelatinous spear, hardens on touch, glossy like glass with zaps of electricity going around it."
+	color = "#00ff15"
+
+/obj/item/spear/slime/throw_impact(atom/hit_atom, datum/thrownthing/throwingdatum)
+	. = ..()
+	if(isliving(hit_atom))
+		playsound(hit_atom, 'sound/effects/glassbr3.ogg', 100)
+		qdel(src)
+
+/obj/item/storage/photo_album/slime
+	name = "slimey photo album"
+	color = "#00ff15"
+
+/obj/item/reagent_containers/food/snacks/pie/plain/slime
+	name = "slime pie"
+	desc = "A slime pie, still delicious."
+	color = "#00ff15"
+
+/obj/item/reagent_containers/food/snacks/pie/plain/slime/On_Consume(mob/living/eater)
+	. = ..()
+	if(isslimeperson(eater))
+		var/mob/living/carbon/human/human_eater = eater
+		human_eater.blood_volume += 50
+
+/obj/machinery/photocopier/slime
+	name = "Slime photocopier"
+	desc = "Used to copy important documents and anatomy studies, it can run on slime jelly!"
+	anchored = FALSE
+	color = "#00ff15"
+
+/obj/machinery/photocopier/slime/attackby(obj/item/O, mob/user, params)
+	. = ..()
+	var/datum/reagents/reggies = O.reagents
+	if(!reggies)
+		return
+	var/datum/reagent/toxin/slimejelly/SJ =  reggies.has_reagent(/datum/reagent/toxin/slimejelly)
+	if(!SJ)
+		return
+	toner += round(SJ.volume/10)
+	reggies.remove_reagent(/datum/reagent/toxin/slimejelly,SJ.volume)
+
+
+/obj/item/slime_iv
+	name = "Slime IV"
+	icon = 'icons/obj/slimecrossing.dmi'
+	icon_state = "slimeiv"
+	var/mob/living/carbon/attached
+
+/obj/item/slime_iv/Initialize()
+	. = ..()
+	create_reagents(100, OPENCONTAINER)
+
+/obj/item/slime_iv/MouseDrop(mob/living/target)
```

Comment:
you might wanna make it also possible to attach this to someone by bopping them with it, with maybe 5u of it injecting instantly (like a hypospray), to reduce jank